packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -97,6 +97,38 @@ describe('cluster', () => {
     });
   });
 
+
+  test('check exposure of EnableIAMDatabaseAuthentication property in DBCluster', () => {
+    // GIVEN
+    const stack = testStack();
+    const vpc = new ec2.Vpc(stack, 'VPC');
+
+    // WHEN
+    new DatabaseCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA,
+      instances: 1,
+      credentials: {
+        username: 'admin',
+        password: cdk.SecretValue.plainText('tooshort'),
+      },
+      instanceProps: {
+        instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.SMALL),
+        vpc,
+      },
+      iamAuthentication: false,
+    });
```

Comment:
Let's remove all of the non-required properties from this test which do not serve any purpose. So, this should be:

```suggestion
    new DatabaseCluster(stack, 'Database', {
      engine: DatabaseClusterEngine.AURORA,
      instanceProps: { vpc },
      iamAuthentication: true,
    });
```